include/seqan/sequence/string_set_dependent_tight.h

```diff
@@ -91,7 +91,7 @@ struct Dependent;
 
 // StringSet with individual sequences in a tight string of string pointers and corr. IDs
 template <typename TString>
-class StringSet<TString, Dependent<Tight> >
+class /*[[deprecated]]*/ StringSet<TString, Dependent<Tight> >
```

Comment:
Hm, this is a difficult situation. Making the default deprecated, but not changing the default seems inconsistent to me :confused: